meta-facebook/mfz-wc/src/platform/plat_class.c

```diff
@@ -50,8 +50,9 @@ uint8_t get_2ou_cardtype()
 
 float get_hsc_type_adc_voltage()
```

Comment:
This function should be deleted since no need for MFZ to check hsc type.

Reply:
Done.